PYME/Acquire/sampleInformation.py

```diff
@@ -520,4 +520,66 @@ def createImage(mdh, slide, comments=''):
     print('FIXME: create database entry for image')
     pass
 
-#MetaDataHandler.provideStartMetadata.append(lambda mdh: getSampleDataFailsafe)
\ No newline at end of file
+#MetaDataHandler.provideStartMetadata.append(lambda mdh: getSampleDataFailsafe)
+
+
+class SimpleSampleInfoPanel(wx.Panel):
+    def __init__(self, parent):
+        """Simple sample info panel to tide over until the SampleInfoDialog is
+        refactored to work independently of a sampleDB database.
+
```

Comment:
```suggestion

TODOS/LIMITATIONS: potentially better addressed by the full refactor rather than by modifying this class, but included here for information / to reinforce that this should ideally be replaced at some point in the future.

 - auto-populate and/or hide acquiring user using logon name on multi-user systems (preference would be to have a config option which can be set on a single user system, but which defaults to false)
 - include info on what is labelled (i.e. Sample.Labelling entries). This is super,super useful to have in the metadata
 - because there is no sanity checking / database backend etc ... you will likely get subtly differing values for acquiring user and creator which will make it hard to search on these if the data ever gets ingested into a database in the future.
 
```